.github/workflows/cicd.yaml

```
@@ -267,4 +267,69 @@ jobs:
         run: |
           bash tests/integration/preconfig-integration-test.sh
         shell: bash
+  
+  upgrade-test:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install yq and bash
+        run: |
+          sudo snap install yq
+          sudo apt update
+          sudo apt install bash -y
+        shell: bash
+      - uses: actions/download-artifact@v2
+        with:
+          name: images
+      - name: Load Docker images
+        run: |
+          docker load -i ${GITHUB_SHA}_image.tar
+        shell: bash
+      - uses: ./.github/actions/k8s-version-config
+        name: Setup k8s cluster
+        with:
+          k8s-version: v1.22
+      - uses: actions/checkout@v2
+        with:
+          ref: 'master'
+      - name: Configure Connaisseur (master)
+        run: |
+          yq e '.detectionMode = "true"' -i helm/values.yaml
```

this will set the `detectionMode` to a string with value `"true"`, NOT a boolean. internally this will still evaluate as `true`, since the string isn't empty, but might cause trouble when changing back.

```suggestion
          yq e '.detectionMode=true' -i helm/values.yaml
```